workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/ReaderWriterInjector.cs

```diff
@@ -10,7 +10,7 @@ namespace Improbable.Gdk.Core.GameObjectRepresentation
     /// <summary>
     ///     Retrieves Reader and Writer fields from MonoBehaviours and handles injection into them.
     /// </summary>
-    internal class SpatialOSBehaviourLibrary
+    internal class ReaderWriterInjector
```

Comment:
The problem with ReaderWriter is that we are going to introduce more for commands. Are those also going to be handled here or elsewhere.

Reply:
That's something I'm planning to think about right now, no plans made yet... But yes, this Injector should almost certainly be generalized to handle those as well, good point